src/codegen/templates/task/_in_stream.py.mako

```diff
@@ -94,13 +96,13 @@ ${property_template.script_property(attribute)}\
             return num_samps_per_chan
 
     def configure_logging(
-            self, file_path, logging_mode=LoggingMode.LOG_AND_READ,
+            self, file_path: Union[str, pathlib.PurePath], logging_mode=LoggingMode.LOG_AND_READ,
             group_name="", operation=LoggingOperation.OPEN_OR_CREATE):
         """
         Configures TDMS file logging for the task.
 
         Args:
-            file_path (str): Specifies the path to the TDMS file to
+            file_path (Union[str, pathlib.PurePath]): Specifies the path to the TDMS file to
```

Comment:
FYI, parameters that have type hints don't need to specify the type in the docstring: https://github.com/ni/nidaqmx-python/pull/539#discussion_r1525339107

Reply:
It's updated as per suggestion. I also updated [function_template.py.mako ](https://github.com/ni/nidaqmx-python/pull/542/files#diff-de3716142179fdfddaef0c1a49af4f98923119b0ed3264e7a17d5da7fe276ed9R40) to get the same effect for functions in physical_channel.py.